Hymns/Tags/TagListView.swift

```diff
@@ -0,0 +1,29 @@
+import SwiftUI
+import Resolver
+
+struct TagListView: View {
```

Comment:
This should probably be in the "Browse" group, not in "Tags"

Reply:
Moded